src/style.css

```diff
@@ -1,3 +1,63 @@
+html {
+  margin: 0;
+  padding: 0;
+  --alternative-text-color: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
```

Comment:
body doesn't have default padding to reset

Reply:
✅